settings.gradle

```diff
@@ -0,0 +1,2 @@
+include ':sdk:core:azure-core'
+include ':sdk:storage:azure-storage-common'
```

Comment:
any advantage in moving `settings.xml` file from `sdk` dir? With this here we have to point AndroidStudio to the root, root contains other artifacts not used for daily dev, wondering whether it affects editor experience.

Reply:
I cloned this PR and opened in AndroidStudio. Though it showed everything when opened first time, after the load/sync there exists only the two modules (core and storage-common). We're good here.